web-marketplace/src/components/organisms/LoginButton/hooks/useSocialProviders.ts

```diff
@@ -13,9 +13,8 @@ export const useSocialProviders = (createProject: boolean = false) => {
         await track<GoogleLoginEvent>('loginGoogle', {
           date: new Date().toUTCString(),
         });
-        window.location.href = `${apiUri}/marketplace/v1/auth/google${
-          createProject ? '?route=project-pages/draft/basic-info' : ''
-        }`;
+        // prettier-ignore
+        window.location.href = `${apiUri}/marketplace/v1/auth/google${redirectRoute ? `?route=${redirectRoute}` : ''}`;
```

Comment:
you might have an issue with your local set up, I don't think we need to prettier ignore here, if I format this locally, I get:

```
window.location.href = `${apiUri}/marketplace/v1/auth/google${
  redirectRoute ? `?route=${redirectRoute}` : ''
}`;
```

which will work

Reply:
That's the same formatting I get with my local set up. Having the string in one line fixed the issue for me.

Reply:
but the formatting I put above works, the one that was wrong because it added some space was:
```
window.location.href = `${apiUri}/marketplace/v1/auth/google
        ${redirectRoute ? `?route=${redirectRoute}` : ''}`;
```
which is different

please let's just use the correct formatting and remove prettier-ignore

Reply:
No idea why the code had that formatting, neither why using `prettier-ignore` is a problem. My editor formats the string literal exactly like yours, so I have pushed it again. Hopefully this time it'll work as expected.